feature/google/src/bc5SupportDev/java/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -922,13 +922,13 @@ class BillingWrapper(
     }
 
     override fun mapStoreProducts(offeringsIn: Offerings, products: List<StoreProduct>): Offerings {
-        if (getIsBC5Enabled(offeringsIn)) {
-            return mapStoreProducts(offeringsIn, products,
+        return if (useBC5) {
```

Comment:
another option would be to have a BC5BillingWrapper that extends from BillingWrapper and only overrides the methods it needs to., like this one